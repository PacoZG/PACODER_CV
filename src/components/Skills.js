import React from 'react'
import { useTranslation } from 'react-i18next'

const Skills = () => {
  const { t } = useTranslation()
  return (
    <div className="p-8 md:p-16 bg-gray-900">
      <div className="flex flex-col justify-center lg:space-x-28 border-double border-b-8 border-gray-400 pb-12">
        <div className="hidden lg:flex flex-col space-y-9">
          <div className="flex items-center justify-center space-x-28">
            <div className="flex flex-col items-center ">
              <img
                className="skill-logo bg-yellow-200 p-2 "
                src="https://img.icons8.com/nolan/96/javascript.png"
                alt="JS"
              />
              <p className="text-yellow-500">Javascript</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-600 p-2  "
                src="https://img.icons8.com/officel/80/000000/react.png"
                alt="React"
              />
              <p className="text-blue-300">React</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-white p-3 "
                src="https://img.icons8.com/color/96/000000/redux.png"
                alt="React"
              />
              <p className="text-purple-800">Redux</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-white p-2 "
                src="https://img.icons8.com/nolan/96/react-native.png"
                alt="React"
              />
              <p className="text-pink-800">React-Native</p>
            </div>
          </div>
          <div className="flex items-center justify-center space-x-24">
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-300 p-2 "
                src="https://img.icons8.com/color/96/000000/mongodb.png"
                alt="React"
              />
              <p className="text-green-400">MongoDB</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-200 p-2 "
                src="https://img.icons8.com/color/48/000000/graphql.png"
                alt="React"
              />
              <p className="text-pink-500">GraphQL</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-green-400 p-2  "
                src="https://img.icons8.com/color/96/000000/nodejs.png"
                alt="React"
              />
              <p className="text-green-400">Node</p>
            </div>
          </div>
          <div className="flex items-center justify-center space-x-24">
            <div className="flex flex-col items-center">
              <img
                className="skill-logo p-3 "
                src="https://tailwindcss.com/_next/static/media/tailwindcss-mark.cb8046c163f77190406dfbf4dec89848.svg"
                alt="React"
              />
              <p className="text-blue-300">Tailwindcss</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo"
                src="https://pics.freeicons.io/uploads/icons/png/3556671901536211770-512.png"
                alt="Cypress.io"
              />
              <p className="text-gray-50">Cypress.io</p>
            </div>
          </div>
        </div>

        <div className="lg:hidden block">
          <div className="flex flex-col items-center space-y-4 sm:space-y-8 md:space-y-12">
            <div className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20">
              <div className="flex flex-col items-center ">
                <img
                  className="skill-logo bg-yellow-200 p-2 "
                  src="https://img.icons8.com/nolan/96/javascript.png"
                  alt="JS"
                />
                <p className="text-sm text-yellow-500">Javascript</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-600 p-2 "
                  src="https://img.icons8.com/officel/80/000000/react.png"
                  alt="React"
                />
                <p className="text-sm text-blue-300">React</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-white p-3 "
                  src="https://img.icons8.com/color/96/000000/redux.png"
                  alt="Redux"
                />
                <p className="text-sm text-purple-800">Redux</p>
              </div>
            </div>

            <div className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20">
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-white p-2 "
                  src="https://img.icons8.com/nolan/96/react-native.png"
                  alt="React"
                />
                <p className="text-sm text-pink-800">React-Native</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-300 p-2 "
                  src="https://img.icons8.com/color/96/000000/mongodb.png"
                  alt="React"
                />
                <p className="text-sm text-green-400">MongoDB</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-200 p-2 "
                  src="https://img.icons8.com/color/48/000000/graphql.png"
                  alt="React"
                />
                <p className="text-sm text-pink-500">GraphQL</p>
              </div>
            </div>

            <div className="flex items-center justify-evenly space-x-5 sm:space-x-11 md:space-x-14">
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-green-400 p-2  "
                  src="https://img.icons8.com/color/96/000000/nodejs.png"
                  alt="React"
                />
                <p className="text-sm text-green-400">Node</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo p-3 "
                  src="https://tailwindcss.com/_next/static/media/tailwindcss-mark.cb8046c163f77190406dfbf4dec89848.svg"
                  alt="React"
                />
                <p className="text-sm text-blue-300">Tailwindcss</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo"
                  src="https://pics.freeicons.io/uploads/icons/png/3556671901536211770-512.png"
                  alt="Cypress.io"
                />
                <p className="text-sm text-gray-300">Cypress.io</p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div className="flex flex-col xl:flex-row items-center space-y-5 xl:space-y-0 p-6 pl-12">
        <div className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full">
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-1/2">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full">
              {t('skills.multitask')}
            </p>
          </div>
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-1/2">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full">
              {t('skills.engaged')}
            </p>
          </div>
        </div>
        <div className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full">
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-1/2">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-gray-300 text-center text-xs sm:text-sm uppercase p-2">
              {t('skills.lang1')}
            </p>
          </div>
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-1/2">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-gray-300 text-center text-xs sm:text-sm uppercase p-2">
              {t('skills.lang2')}
            </p>
          </div>
        </div>
      </div>
    </div>
  )
}

export default Skills
