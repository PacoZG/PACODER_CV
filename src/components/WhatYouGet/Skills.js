import React from 'react'
import { useTranslation } from 'react-i18next'
import { ReactComponent as Cypress } from '../../utils/assets/cypress.svg'
import { ReactComponent as Typescript } from '../../utils/assets/typescript.svg'
import { ReactComponent as AWSLogo } from '../../utils/assets/aws-logo.svg'
import Tailwind from '../../utils/assets/tailwind-css.png'
const Skills = () => {
  const { t } = useTranslation()
  return (
    <div className="flex flex-col items-center lg:gap-8 justify-center p-8 md:p-16 bg-gray-300 dark:bg-gray-900 transition duration-500 bg-opacity-70 dark:bg-opacity-70">
      <div className="flex flex-col justify-center lg:space-x-28 border-double border-b-8 border-gray-700 dark:border-gray-500  pb-12 lg:w-10/12">
        <div className="hidden lg:flex flex-col space-y-9">
          <div className="flex items-center justify-center gap-24">
            <div className="flex flex-col items-center ">
              <img
                className="skill-logo bg-yellow-200 p-2 "
                src="https://img.icons8.com/nolan/96/javascript.png"
                alt="JS"
              />
              <p className="text-black dark:text-yellow-500 transition duration-500">Javascript</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-600 p-2  "
                src="https://img.icons8.com/officel/80/000000/react.png"
                alt="React"
              />
              <p className="text-black dark:text-blue-300 transition duration-500">React</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-white p-3 "
                src="https://img.icons8.com/color/96/000000/redux.png"
                alt="React"
              />
              <p className="text-black dark:text-purple-800 transition duration-500">Redux</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-white p-2 "
                src="https://img.icons8.com/nolan/96/react-native.png"
                alt="React"
              />
              <p className="text-black dark:text-pink-800 transition duration-500">React-Native</p>
            </div>
          </div>
          <div className="flex items-center justify-center gap-24">
            <div className="flex flex-col items-center ">
              <Typescript className=" skill-logo w-20 rounded-none  " />
              <p className="text-black dark:text-yellow-500 transition duration-500">Typescript</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-300 p-2 "
                src="https://img.icons8.com/color/96/000000/mongodb.png"
                alt="React"
              />
              <p className="text-black dark:text-green-400 transition duration-500">MongoDB</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-gray-200 p-2 "
                src="https://img.icons8.com/color/48/000000/graphql.png"
                alt="React"
              />
              <p className="text-black dark:text-pink-500 transition duration-500">GraphQL</p>
            </div>
            <div className="flex flex-col items-center">
              <img
                className="skill-logo bg-green-400 p-2  "
                src="https://img.icons8.com/color/96/000000/nodejs.png"
                alt="React"
              />
              <p className="text-black dark:text-green-400 transition duration-500">Node</p>
            </div>
          </div>
          <div className="flex items-center justify-center gap-24">
            <div className="flex flex-col items-center">
              <img alt="tailwind-logo" src={Tailwind} className="skill-logo mb-4 h-20 w-32" />
              <p className="text-black dark:text-blue-300 transition duration-500">Tailwindcss</p>
            </div>
            <div className="flex flex-col items-center">
              <Cypress className="skill-logo h-16 sm:h-20 md:h-24" />
              <p className="text-black dark:text-gray-50 transition duration-500">Cypress.io</p>
            </div>
            <div className="flex flex-col items-center">
              <AWSLogo className="skill-logo p-3 w-24 h-24 bg-white" />
              <p className=" text-yellow-400 transition duration-500">AWS Junior Developer Use</p>
            </div>
          </div>
        </div>

        <div className="lg:hidden block">
          <div className="flex flex-col items-center space-y-4 sm:space-y-8 md:space-y-12">
            <div className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20">
              <div className="flex flex-col items-center ">
                <img
                  className="skill-logo bg-yellow-200 p-2 "
                  src="https://img.icons8.com/nolan/96/javascript.png"
                  alt="JS"
                />
                <p className="text-sm text-black dark:text-yellow-500 transition duration-500">Javascript</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-600 p-2 "
                  src="https://img.icons8.com/officel/80/000000/react.png"
                  alt="React"
                />
                <p className="text-sm text-black dark:text-blue-300 transition duration-500">React</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-white p-3 "
                  src="https://img.icons8.com/color/96/000000/redux.png"
                  alt="Redux"
                />
                <p className="text-sm text-black dark:text-purple-800 transition duration-500">Redux</p>
              </div>
            </div>

            <div className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20">
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-white p-2 "
                  src="https://img.icons8.com/nolan/96/react-native.png"
                  alt="React"
                />
                <p className="text-sm text-black dark:text-pink-800 transition duration-500">React-Native</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-300 p-2 "
                  src="https://img.icons8.com/color/96/000000/mongodb.png"
                  alt="React"
                />
                <p className="text-sm text-black dark:text-green-400 transition duration-500">MongoDB</p>
              </div>
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-gray-200 p-2 "
                  src="https://img.icons8.com/color/48/000000/graphql.png"
                  alt="React"
                />
                <p className="text-sm text-black dark:text-pink-500 transition duration-500">GraphQL</p>
              </div>
            </div>

            <div className="flex items-center justify-evenly space-x-5 sm:space-x-11 md:space-x-14">
              <div className="flex flex-col items-center">
                <img
                  className="skill-logo bg-green-400 p-2  "
                  src="https://img.icons8.com/color/96/000000/nodejs.png"
                  alt="React"
                />
                <p className="text-sm text-black dark:text-green-400 transition duration-500">Node</p>
              </div>
              <div className="flex flex-col items-center">
                <img alt="tailwind-logo" src={Tailwind} className="skill-logo h-16 w-20 mb-4 pt-3" />
                <p className="text-sm text-black dark:text-blue-300 transition duration-500">Tailwindcss</p>
              </div>
              <div className="flex flex-col items-center">
                <Cypress className="skill-logo h-20 w-20" />
                <p className="text-sm text-black dark:text-gray-300 transition duration-500">Cypress.io</p>
              </div>
            </div>

            <div className="flex items-center justify-evenly space-x-5 sm:space-x-11 md:space-x-14">
              <div className="flex flex-col items-center">
                <Typescript className="skill-logo w-16 rounded-none" />
                <p className="text-sm text-black dark:text-green-400 transition duration-500">Typescript</p>
              </div>
              <div className="flex flex-col items-center">
                <AWSLogo className="skill-logo p-2 my-2 h-16 w-16 bg-white" />
                <p className=" text-yellow-400 transition duration-500">Junior Dev Use</p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div className="flex flex-col xl:flex-row items-center space-y-5 xl:space-y-0 p-6 pl-12">
        <div className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full">
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-max">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full transition duration-500">
              {t('skills.multitask')}
            </p>
          </div>
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-max">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full transition duration-500">
              {t('skills.engaged')}
            </p>
          </div>
        </div>
        <div className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full">
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-max">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 transition duration-500">
              {t('skills.lang1')}
            </p>
          </div>
          <div className="relative flex items-center justify-evenly space-x-7 w-full md:w-max">
            <span className="spinner"></span>
            <p className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 transition duration-500">
              {t('skills.lang2')}
            </p>
          </div>
        </div>
      </div>
    </div>
  )
}

export default Skills
