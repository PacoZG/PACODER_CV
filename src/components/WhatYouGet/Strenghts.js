import React, { useState } from 'react'
import { useTranslation } from 'react-i18next'
import { Transition } from '@headlessui/react'
import { ReactComponent as Arrow } from '../../utils/assets/arrow.svg'

const Strenghts = () => {
  const { t } = useTranslation()
  const [stgr1, setStgr1] = useState(false)
  const [stgr2, setStgr2] = useState(false)
  const [stgr3, setStgr3] = useState(false)
  const [stgr4, setStgr4] = useState(false)
  return (
    <div className="bg-gray-300 dark:bg-gray-900 flex flex-col items-center p-4 md:py-10 lg:py-20 transition duration-500 rounded-b-xl">
      <div className="w-full sm:w-11/12 lg:w-2/3 text-center">
        <div className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3">
          <button className="flex items-center justify-between w-full" onClick={() => setStgr1(!stgr1)}>
            <p className="text-2xl text-gray-900 text-left cursor-pointer pb-2 w-full ">{t('strengths.strength1')}</p>
            <div className={stgr1 ? 'transition duration-300 transform rotate-90 ' : 'transition duration-300'}>
              <Arrow />
            </div>
          </button>
          <Transition
            show={stgr1}
            enter="transition-opacity duration-75"
            enterFrom="opacity-0"
            enterTo="opacity-100"
            leave="transition-opacity duration-150"
            leaveFrom="opacity-100"
            leaveTo="opacity-0"
          >
            <div className="text-gray-300 text-left border-t-2 border-gray-300 pt-2">
              {t('strengths.line1_1')}{' '}
              <a
                className="text-gray-300 underline transition duration-200 hover:text-indigo-600"
                href="https://fullstackopen.com/en/"
                alt="course"
              >
                {t('strengths.line1_2')}
              </a>{' '}
              {t('strengths.line1_3')}
            </div>
          </Transition>
        </div>
        <div className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3">
          <button className="flex items-center justify-between w-full" onClick={() => setStgr2(!stgr2)}>
            <p className="text-2xl text-gray-900 text-left cursor-pointer pb-2 w-full ">{t('strengths.strength2')}</p>
            <div className={stgr2 ? 'transition duration-300 transform rotate-90 ' : 'transition duration-300'}>
              <Arrow />
            </div>
          </button>
          <Transition
            show={stgr2}
            enter="transition-opacity duration-75"
            enterFrom="opacity-0"
            enterTo="opacity-100"
            leave="transition-opacity duration-150"
            leaveFrom="opacity-100"
            leaveTo="opacity-0"
          >
            <p className=" text-gray-300 text-left border-t-2 border-gray-300 pt-2 w-full ">{t('strengths.line2')}</p>
          </Transition>
        </div>
        <div className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3">
          <button className="flex items-center justify-between w-full" onClick={() => setStgr3(!stgr3)}>
            <p className="text-2xl text-gray-900 text-left cursor-pointer pb-2">{t('strengths.strength3')}</p>
            <div className={stgr3 ? 'transition duration-300 transform rotate-90 ' : 'transition duration-300'}>
              <Arrow />
            </div>
          </button>
          <Transition
            show={stgr3}
            enter="transition-opacity duration-75"
            enterFrom="opacity-0"
            enterTo="opacity-100"
            leave="transition-opacity duration-150"
            leaveFrom="opacity-100"
            leaveTo="opacity-0"
          >
            <p className="text-gray-300 text-left border-t-2 border-gray-300 pt-2 w-full ">{t('strengths.line3')}</p>
          </Transition>
        </div>
        <div className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3">
          <button className="flex items-center justify-between w-full" onClick={() => setStgr4(!stgr4)}>
            <p className="text-2xl text-gray-900 text-left cursor-pointer pb-2">
              HIGH <span className="border-2 border-yellow-300 p-1">5</span> TEST
            </p>
            <div className={stgr4 ? 'transition duration-300 transform rotate-90 ' : 'transition duration-300'}>
              <Arrow />
            </div>
          </button>
          <Transition
            show={stgr4}
            enter="transition-opacity duration-75"
            enterFrom="opacity-0"
            enterTo="opacity-100"
            leave="transition-opacity duration-150"
            leaveFrom="opacity-100"
            leaveTo="opacity-0"
          >
            <p className="text-gray-300 text-left border-t-2 border-gray-300 pt-2">
              {t('strengths.line4_1')}
              <a
                className="text-gray-300 underline transition duration-200 hover:text-indigo-600"
                href="https://high5test.com/test/result-your-friend/MTUwMTU5Mg==/"
                target="blank"
              >
                {t('strengths.line4_2')}
              </a>
              {t('strengths.line4_3')}
            </p>
          </Transition>
        </div>
      </div>
    </div>
  )
}

export default Strenghts
