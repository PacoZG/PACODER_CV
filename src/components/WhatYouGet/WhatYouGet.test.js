import React from 'react'
import { I18nextProvider } from 'react-i18next'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import i18n from '../../utils/language/i18nextForTest'
import Skills from './Skills'
import Strenghts from './Strenghts'

describe('Testing Working skill and experience page', () => {
  test('Snapshot of the Skills component', () => {
    const redered = renderer
      .create(
        <BrowserRouter>
          <I18nextProvider i18n={i18n}>
            <Skills />
          </I18nextProvider>
        </BrowserRouter>
      )
      .toJSON()
    expect(redered).toMatchSnapshot()
  })

  test('Snapshot of the Strenghts component', () => {
    const redered = renderer
      .create(
        <BrowserRouter>
          <I18nextProvider i18n={i18n}>
            <Strenghts />
          </I18nextProvider>
        </BrowserRouter>
      )
      .toJSON()
    expect(redered).toMatchSnapshot()
  })
})
