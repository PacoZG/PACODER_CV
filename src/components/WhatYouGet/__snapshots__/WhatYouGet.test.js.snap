// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing header Snapshot of the Skills component 1`] = `
<div
  className="flex flex-col items-center lg:gap-8 justify-center p-8 md:p-16 bg-gray-300 dark:bg-gray-900 transition duration-500 bg-opacity-70 dark:bg-opacity-70"
>
  <div
    className="flex flex-col justify-center lg:space-x-28 border-double border-b-8 border-gray-700 dark:border-gray-500  pb-12 lg:w-10/12"
  >
    <div
      className="hidden lg:flex flex-col space-y-9"
    >
      <div
        className="flex items-center justify-center gap-24"
      >
        <div
          className="flex flex-col items-center "
        >
          <img
            alt="JS"
            className="skill-logo bg-yellow-200 p-2 "
            src="https://img.icons8.com/nolan/96/javascript.png"
          />
          <p
            className="text-black dark:text-yellow-500 transition duration-500"
          >
            Javascript
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-gray-600 p-2  "
            src="https://img.icons8.com/officel/80/000000/react.png"
          />
          <p
            className="text-black dark:text-blue-300 transition duration-500"
          >
            React
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-white p-3 "
            src="https://img.icons8.com/color/96/000000/redux.png"
          />
          <p
            className="text-black dark:text-purple-800 transition duration-500"
          >
            Redux
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-white p-2 "
            src="https://img.icons8.com/nolan/96/react-native.png"
          />
          <p
            className="text-black dark:text-pink-800 transition duration-500"
          >
            React-Native
          </p>
        </div>
      </div>
      <div
        className="flex items-center justify-center gap-24"
      >
        <div
          className="flex flex-col items-center "
        >
          <svg
            className=" skill-logo w-20 rounded-none  "
          >
            typescript.svg
          </svg>
          <p
            className="text-black dark:text-yellow-500 transition duration-500"
          >
            Typescript
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-gray-300 p-2 "
            src="https://img.icons8.com/color/96/000000/mongodb.png"
          />
          <p
            className="text-black dark:text-green-400 transition duration-500"
          >
            MongoDB
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-gray-200 p-2 "
            src="https://img.icons8.com/color/48/000000/graphql.png"
          />
          <p
            className="text-black dark:text-pink-500 transition duration-500"
          >
            GraphQL
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="React"
            className="skill-logo bg-green-400 p-2  "
            src="https://img.icons8.com/color/96/000000/nodejs.png"
          />
          <p
            className="text-black dark:text-green-400 transition duration-500"
          >
            Node
          </p>
        </div>
      </div>
      <div
        className="flex items-center justify-center gap-24"
      >
        <div
          className="flex flex-col items-center"
        >
          <img
            alt="tailwind-logo"
            className="skill-logo mb-4 h-20 w-32"
            src="tailwind-css.png"
          />
          <p
            className="text-black dark:text-blue-300 transition duration-500"
          >
            Tailwindcss
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <svg
            className="skill-logo h-16 sm:h-20 md:h-24"
          >
            cypress.svg
          </svg>
          <p
            className="text-black dark:text-gray-50 transition duration-500"
          >
            Cypress.io
          </p>
        </div>
        <div
          className="flex flex-col items-center"
        >
          <svg
            className="skill-logo p-3 w-24 h-24 bg-white"
          >
            aws-logo.svg
          </svg>
          <p
            className=" text-yellow-400 transition duration-500"
          >
            AWS Junior Developer Use
          </p>
        </div>
      </div>
    </div>
    <div
      className="lg:hidden block"
    >
      <div
        className="flex flex-col items-center space-y-4 sm:space-y-8 md:space-y-12"
      >
        <div
          className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20"
        >
          <div
            className="flex flex-col items-center "
          >
            <img
              alt="JS"
              className="skill-logo bg-yellow-200 p-2 "
              src="https://img.icons8.com/nolan/96/javascript.png"
            />
            <p
              className="text-sm text-black dark:text-yellow-500 transition duration-500"
            >
              Javascript
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="React"
              className="skill-logo bg-gray-600 p-2 "
              src="https://img.icons8.com/officel/80/000000/react.png"
            />
            <p
              className="text-sm text-black dark:text-blue-300 transition duration-500"
            >
              React
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="Redux"
              className="skill-logo bg-white p-3 "
              src="https://img.icons8.com/color/96/000000/redux.png"
            />
            <p
              className="text-sm text-black dark:text-purple-800 transition duration-500"
            >
              Redux
            </p>
          </div>
        </div>
        <div
          className="flex items-center justify-evenly space-x-9 sm:space-x-16 md:space-x-20"
        >
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="React"
              className="skill-logo bg-white p-2 "
              src="https://img.icons8.com/nolan/96/react-native.png"
            />
            <p
              className="text-sm text-black dark:text-pink-800 transition duration-500"
            >
              React-Native
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="React"
              className="skill-logo bg-gray-300 p-2 "
              src="https://img.icons8.com/color/96/000000/mongodb.png"
            />
            <p
              className="text-sm text-black dark:text-green-400 transition duration-500"
            >
              MongoDB
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="React"
              className="skill-logo bg-gray-200 p-2 "
              src="https://img.icons8.com/color/48/000000/graphql.png"
            />
            <p
              className="text-sm text-black dark:text-pink-500 transition duration-500"
            >
              GraphQL
            </p>
          </div>
        </div>
        <div
          className="flex items-center justify-evenly space-x-5 sm:space-x-11 md:space-x-14"
        >
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="React"
              className="skill-logo bg-green-400 p-2  "
              src="https://img.icons8.com/color/96/000000/nodejs.png"
            />
            <p
              className="text-sm text-black dark:text-green-400 transition duration-500"
            >
              Node
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <img
              alt="tailwind-logo"
              className="skill-logo h-16 w-20 mb-4 pt-3"
              src="tailwind-css.png"
            />
            <p
              className="text-sm text-black dark:text-blue-300 transition duration-500"
            >
              Tailwindcss
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <svg
              className="skill-logo h-20 w-20"
            >
              cypress.svg
            </svg>
            <p
              className="text-sm text-black dark:text-gray-300 transition duration-500"
            >
              Cypress.io
            </p>
          </div>
        </div>
        <div
          className="flex items-center justify-evenly space-x-5 sm:space-x-11 md:space-x-14"
        >
          <div
            className="flex flex-col items-center"
          >
            <svg
              className="skill-logo w-16 rounded-none"
            >
              typescript.svg
            </svg>
            <p
              className="text-sm text-black dark:text-green-400 transition duration-500"
            >
              Typescript
            </p>
          </div>
          <div
            className="flex flex-col items-center"
          >
            <svg
              className="skill-logo p-2 my-2 h-16 w-16 bg-white"
            >
              aws-logo.svg
            </svg>
            <p
              className=" text-yellow-400 transition duration-500"
            >
              Junior Dev Use
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex flex-col xl:flex-row items-center space-y-5 xl:space-y-0 p-6 pl-12"
  >
    <div
      className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full"
    >
      <div
        className="relative flex items-center justify-evenly space-x-7 w-full md:w-max"
      >
        <span
          className="spinner"
        />
        <p
          className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full transition duration-500"
        >
          Ability to multitask
        </p>
      </div>
      <div
        className="relative flex items-center justify-evenly space-x-7 w-full md:w-max"
      >
        <span
          className="spinner"
        />
        <p
          className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 w-full transition duration-500"
        >
          Engaged, persistent and focused
        </p>
      </div>
    </div>
    <div
      className="flex flex-col md:flex-row items-center space-y-5 md:space-y-0 w-full"
    >
      <div
        className="relative flex items-center justify-evenly space-x-7 w-full md:w-max"
      >
        <span
          className="spinner"
        />
        <p
          className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 transition duration-500"
        >
          Mother tongue: Spanish
        </p>
      </div>
      <div
        className="relative flex items-center justify-evenly space-x-7 w-full md:w-max"
      >
        <span
          className="spinner"
        />
        <p
          className="md:px-5 my-2 text-black dark:text-gray-300 text-center text-xs sm:text-sm uppercase p-2 transition duration-500"
        >
          Proficiency in English
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`Testing header Snapshot of the Strenghts component 1`] = `
<div
  className="bg-gray-300 dark:bg-gray-900 flex flex-col items-center p-4 md:py-10 lg:py-20 transition duration-500 rounded-b-xl bg-opacity-70 dark:bg-opacity-70"
>
  <div
    className="w-full sm:w-11/12 lg:w-2/3 text-center"
  >
    <div
      className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3"
    >
      <button
        className="flex items-center justify-between w-full"
        onClick={[Function]}
      >
        <p
          className="text-2xl text-gray-900 text-left cursor-pointer pb-2 w-full "
        >
          Committed and Perseverant
        </p>
        <div
          className="transition duration-300"
        >
          <svg>
            arrow.svg
          </svg>
        </div>
      </button>
    </div>
    <div
      className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3"
    >
      <button
        className="flex items-center justify-between w-full"
        onClick={[Function]}
      >
        <p
          className="text-2xl text-gray-900 text-left cursor-pointer pb-2 w-full "
        >
          Customer Service Oriented
        </p>
        <div
          className="transition duration-300"
        >
          <svg>
            arrow.svg
          </svg>
        </div>
      </button>
    </div>
    <div
      className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3"
    >
      <button
        className="flex items-center justify-between w-full"
        onClick={[Function]}
      >
        <p
          className="text-2xl text-gray-900 text-left cursor-pointer pb-2"
        >
          Teaching Experience
        </p>
        <div
          className="transition duration-300"
        >
          <svg>
            arrow.svg
          </svg>
        </div>
      </button>
    </div>
    <div
      className="p-4 border-2 border-gray-300 rounded-md bg-gray-500 mb-3"
    >
      <button
        className="flex items-center justify-between w-full"
        onClick={[Function]}
      >
        <p
          className="text-2xl text-gray-900 text-left cursor-pointer pb-2"
        >
          HIGH 
          <span
            className="border-2 border-yellow-300 p-1"
          >
            5
          </span>
           TEST
        </p>
        <div
          className="transition duration-300"
        >
          <svg>
            arrow.svg
          </svg>
        </div>
      </button>
    </div>
  </div>
</div>
`;
